media/js/analytics.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+var dntEnabled = require("@mozmeao/dnt-helper");
```

Comment:
You need to include the dnt-helper script. Copy/paste it in like we do on blogs or work some npm magic.

Reply:
hah, turns out it's already a dependency because this project started as a copy of /openletter and I never removed that bit 🫠 
https://github.com/mozilla/rise25/blob/main/package.json#L35